seL4-platforms/platforms.yml

```diff
@@ -118,6 +118,9 @@ platforms:
     platform: maaxboard
     req: [maaxboard1]
     march: armv8a
+    extra_settings:
```

Comment:
Maybe we should pick a more specific name like `build_params` here. It might also be worth having a note here why these setting are needed and the defaults are not used. Would it make sense to change the defaults instead. This platform is quite new, so there might no be too much dependencies on the default.